src/tools/gophish_import.py

```diff
@@ -117,23 +117,24 @@ def load_groups(api, assessment):
                 break
             except Error as e:
                 if e.message == "Group name already in use":
-                    logging.warning(f"{e}. Finding previously loaded group to delete.")
+                    logging.warning("%s. Finding previously loaded group to delete.", e)
                     groups = api.groups.get()
                     logging.debug(
-                        f"Checking {len(groups)} for previously imported group to get ID"
+                        "Checking %i for previously imported group to get ID",
+                        len(groups),
                     )
                     for old_group in groups:
                         if old_group.name == new_group.name:
-                            logging.debug(f"Deleting Group with ID {old_group.id}")
+                            logging.debug("Deleting Group with ID %i", old_group.id)
                             api.groups.delete(old_group.id)
                             logging.info("Re-Loading new group.")
                 else:
-                    logging.error(f"{e}\n")
+                    logging.error("%s\n", e)
                     raise
 
         group["id"] = new_group.id
 
-        logging.info("Group Ready: {}\n".format(new_group.name))
+        logging.info("Group Ready: %s\n", new_group.name)
```

Comment:
Why is a trailing `\n` required here and just above?  I don't think you do that anywhere else in the code.  I suggest removing it if there isn't a specific reason for it.